project-standards/technical-agility/up-to-date-dependencies.s.md

```diff
@@ -19,3 +19,26 @@
 
 ## Good Examples
 *TBD*
+
+## Upgrading React Native
+
+React Native is very probably the dependency you'll have to upgrade the more often so we felt it was worth dedicating a whole section to it.
+
+### How to
+
+First of all, if you are late by several versions do them one at a time.
+Skipping one might be really tough and you can get stuff done more often if you do it one version at a time.
```

Comment:
"Upgrading several version at a time (like from 0.42 to 0.46) can create a **lot** of complication, because React Native changes many javascript and native dependencies that also impact your code and its structure. If you do one version at a time, you are sure to be able to more quickly give a working version with a slightly newer version (like from 0.42 to 0.43, etc.) that you tested on both Android and iOS."